doc/source/manual/xtrabackup_sst.rst

```diff
@@ -163,14 +170,15 @@ if :option:`encrypt` is set to ``1``.
 
 .. option:: sockopt
 
-Used to specify key/value pairs of socket options, separated by commas,
-for example::
+Used to specify key/value pairs of socket options, separated by commas. For example, `retry=`N, then the socat retries the N number of times. If no value is specified, the default is 30. 
```

Comment:
I would change the last sentence to something like:
If no `retry=N` is specified, SST script implicitly adds `retry=30` to `sockopt`